<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ICP project 2020 - traffic tracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>730</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="exit_button">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>43</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>43</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>43</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>43</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>43</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>43</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>43</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>43</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>43</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 43, 39)</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1111</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_sim">
     <attribute name="title">
      <string>Simulated</string>
     </attribute>
     <widget class="QPushButton" name="density_button">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>620</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Traffic Density</string>
      </property>
     </widget>
     <widget class="MapRenderer" name="graphicsViewSim">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1101</width>
        <height>541</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>50</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Set Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Set Speed</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTimeEdit" name="timeEdit">
           <property name="displayFormat">
            <string>hh:mm:ss</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>120.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>0</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="detour_button">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>560</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Detour</string>
      </property>
     </widget>
     <widget class="RendererLinkOverview" name="graphicsViewLink">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>560</y>
        <width>961</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="detour_reset_button">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>500</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset All Detours</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_real">
     <attribute name="title">
      <string>Real</string>
     </attribute>
     <widget class="QLCDNumber" name="lcdReal">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>0</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="MapRenderer" name="graphicsViewReal">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1101</width>
        <height>681</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdRealMiami">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>50</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QLabel" name="info_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>610</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>graphicsViewReal</zorder>
     <zorder>lcdReal</zorder>
     <zorder>lcdRealMiami</zorder>
     <zorder>info_label</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1148</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RendererLinkOverview</class>
   <extends>QGraphicsView</extends>
   <header>rendererlinkoverview.h</header>
  </customwidget>
  <customwidget>
   <class>MapRenderer</class>
   <extends>QGraphicsView</extends>
   <header>maprenderer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <signal>signal1()</signal>
  <slot>slot1()</slot>
 </slots>
</ui>
